docs/docsite/Makefile

```diff
@@ -69,6 +69,8 @@ clean:
 	-rm rst/config.rst
 	-rm rst/ansible-*.rst
```

Comment:
This needs more work, see my comment on your other PR

Most of these other `rm`'s` need updating to add the new paths in